<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
    SPDX-FileCopyrightText: none
    SPDX-License-Identifier: GPL-3.0-or-later
 </author>
 <class>VideoImportDialog</class>
 <widget class="QWidget" name="VideoImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>559</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Video Animation Import</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="fileLocationLabel">
         <property name="text">
          <string>Video file:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="KisWarningBlock" name="lblWarning"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="thumbnailImageHolder">
               <property name="minimumSize">
                <size>
                 <width>320</width>
                 <height>240</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="text">
                <string>No Preview</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QSlider" name="videoPreviewSlider">
                  <property name="maximum">
                   <number>1</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="videoPreviewSliderValueLabel">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>50</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>0</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Metadata</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="fileLoadedDetails">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="text">
                <string>Please choose a file.</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="wordWrap">
                <bool>true</bool>
               </property>
               <property name="margin">
                <number>8</number>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QPushButton" name="prevFrameButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="currentFrameNumberInput">
                 <property name="maximumSize">
                  <size>
                   <width>96</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                 <property name="maximum">
                  <number>99999</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="nextFrameButton">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>32</height>
                  </size>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QGroupBox" name="exportOptionsGroup">
         <property name="title">
          <string>Import Options</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="1" column="0" colspan="2">
           <widget class="QGroupBox" name="optionFilterDuplicates">
            <property name="toolTip">
             <string>Use ffmpeg to detect similar frames and reduce the total number of frames imported.</string>
            </property>
            <property name="title">
             <string>Filter duplicate frames</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::ExpandingFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="sensitivityLabel">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Sensitivity</string>
               </property>
               <property name="indent">
                <number>-1</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="KisDoubleSliderSpinBox" name="sensitivitySpinbox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="toolTip">
                <string>How sensitive should the duplicate frame filter be when determining
which frames are added to Krita.
The higher the sensitivity, the more frames are added to Krita.</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="importDocumentLabel">
            <property name="text">
             <string>Import into:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cmbDocumentHandler">
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="2">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_4" native="true">
               <layout class="QFormLayout" name="formLayout_7">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="fpsLabel">
                  <property name="text">
                   <string>FPS:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="fpsSpinbox"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="frameSkipLabel">
                  <property name="text">
                   <string>Skip Interval:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="frameSkipSpinbox"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QFormLayout" name="formLayout_9">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0" colspan="2">
                 <widget class="QLabel" name="fpsDocumentLabel">
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QFormLayout" name="formLayout_8">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="startExportingAtLabel">
                  <property name="text">
                   <string>Start at: </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QDoubleSpinBox" name="startExportingAtSpinbox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="exportDurationLabel">
                  <property name="text">
                   <string>Duration:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QDoubleSpinBox" name="exportDurationSpinbox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="KisFileNameRequester" name="fileLocation" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOptions">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_6">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="optionsDocumentGroup">
         <property name="title">
          <string>Document</string>
         </property>
         <layout class="QFormLayout" name="formLayout1">
          <item row="0" column="0">
           <widget class="QLabel" name="documentWidthLabel">
            <property name="text">
             <string>Width:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="documentWidthSpinbox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="documentHeightLabel">
            <property name="text">
             <string>Height:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="documentHeightSpinbox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="optionsImportVideoScaleGroup">
         <property name="title">
          <string>Import Video Scale</string>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="videoWidthLabel">
            <property name="text">
             <string>Width:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="videoWidthSpinbox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="videoHeightLabel">
            <property name="text">
             <string>Height:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="videoHeightSpinbox"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="videoScaleFilterLabel">
            <property name="text">
             <string comment="Resize interpolation method list label">Filter:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="cmbVideoScaleFilter"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="optionsFFMpegGroup">
         <property name="title">
          <string>FFmpeg</string>
         </property>
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="ffmpegLocationLabel">
            <property name="text">
             <string>FFmpeg path:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QComboBox" name="cmbFFMpegLocation"/>
            </item>
            <item>
             <widget class="QToolButton" name="ffmpegPickerButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="ffprobeLocationLabel">
            <property name="text">
             <string>FFprobe path:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="cmbFFProbeLocation"/>
            </item>
            <item>
             <widget class="QToolButton" name="ffprobePickerButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">kis_slider_spin_box.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisWarningBlock</class>
   <extends>QGroupBox</extends>
   <header location="global">kis_warning_block.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisFileNameRequester</class>
   <extends>QWidget</extends>
   <header location="global">kis_file_name_requester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>videoPreviewSlider</tabstop>
  <tabstop>prevFrameButton</tabstop>
  <tabstop>currentFrameNumberInput</tabstop>
  <tabstop>nextFrameButton</tabstop>
  <tabstop>fpsSpinbox</tabstop>
  <tabstop>frameSkipSpinbox</tabstop>
  <tabstop>startExportingAtSpinbox</tabstop>
  <tabstop>exportDurationSpinbox</tabstop>
  <tabstop>optionFilterDuplicates</tabstop>
  <tabstop>sensitivitySpinbox</tabstop>
  <tabstop>cmbDocumentHandler</tabstop>
  <tabstop>documentWidthSpinbox</tabstop>
  <tabstop>documentHeightSpinbox</tabstop>
  <tabstop>videoWidthSpinbox</tabstop>
  <tabstop>videoHeightSpinbox</tabstop>
  <tabstop>cmbVideoScaleFilter</tabstop>
  <tabstop>cmbFFMpegLocation</tabstop>
  <tabstop>ffmpegPickerButton</tabstop>
  <tabstop>cmbFFProbeLocation</tabstop>
  <tabstop>ffprobePickerButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
